fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Request.kt

```diff
@@ -90,10 +92,26 @@ class Request(
         return this
     }
 
+    /**
+     *  <p> Note that your value, will be converted to a String via the toString() </p>
+     *  <p> Please note that header of the same key are supported and headers with the same key
+     *  will be sent in the format of `key` : [ value,value,value ] </p>
+     * @param pairs This all the key value pair you wish to add to the headers
+     *
+     * @return the request supplied
+     *
+     * */
     fun header(vararg pairs: Pair<String, Any>?): Request {
-        pairs.forEach {
-            if (it != null)
+        pairs.filterNotNull().forEach {
```

Comment:
You might wanna do `forEach { (key, value) ->` so you don't have to keep calling `.first` or `.second`